platform/core/src/utils/subscribeToNextViewportGridChange.ts

```diff
@@ -24,10 +24,6 @@ function subscribeToNextViewportGridChange(
         viewportGridService.EVENTS.LAYOUT_CHANGED,
         callback
       ),
-      viewportGridService.subscribe(
```

Comment:
I am a little bit concerned with removing this. At present it seems ok, but I suspect at some point there may be a scenario where the grid state changes and the one-up state sync becomes stale. Furthermore, perhaps this utility function should take a list of events instead of hardcoding them. Thoughts?

Reply:
i would worry about it at some point where for some scenarios (that we don't know now) would break.